Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -55,6 +59,14 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         XCTAssertFalse(viewController.notificationFired)
     }
 
+    func testInstantiate() {
+        let bar = "Bar"
+        let myViewController = MyViewController.instantiate(from: "TestStoryboard")
+        XCTAssert(myViewController is MyViewController)
+        myViewController.foo = bar
+        XCTAssertEqual(myViewController.foo, bar)
```

Comment:
Still not sure what tests make sense here. @LucianoPAlmeida @omaralbeik any suggestions?

Reply:
I think we shouldn't assert anything, the test would fail if it crashes... so we should just call the function inside the test and expect the function to return successfully. 
IMO we don't need to assert that :)